daemon/razer_daemon/daemon.py

```diff
@@ -379,43 +413,48 @@ def _remove_devices(self):
             self.logger.warning("Device %s is missing. Removing from DBus", device_id)
             del self._razer_devices[device_id]
 
+    def _udev_input_event(self, device):
+        self.logger.debug('Device event [{0.action}]: {0.device_path}'.format(device))
```

Comment:
probably want to also then filter by something razer specific to avoid unnecessary work. Somthing like if Razer is in /dev/input/by-id path or using their USB VID